Keas.Core/Domain/Key.cs

```diff
@@ -19,6 +19,7 @@ public class Key : AssetBase
         protected internal static void OnModelCreating(ModelBuilder builder)
         {
             builder.Entity<Key>().HasQueryFilter(a => a.Active);
+            builder.Entity<Key>().HasIndex(x => x.Code).IsUnique();

```

Comment:
@srkirkland @jSylvestre does this count as a db change?

Reply:
If we do this, and someone deletes a key, does that mean they can not recreate it if the deletion needs to be reversed? We don't currently have a way to move !active => active. Current accepted method was go create a new one.

Reply:
@laholstege Yes, I think so. And no, we can't do it. (Unique within team and when active)

Reply:
@laholstege yea i think @jcubbage is right and we can't do this as a unique index.  It would mean that a given code could only be used by one team, ever, and they wouldn't be able to inactivate it.   Instead you should get rid of this and just do a SQL check in code to make sure it's unique among active keys.